specs/utils.spec.cjs

```diff
@@ -32,8 +32,8 @@ describe("Utils", () => {
 		});
 
 		it("should convert a Date object to a valid W3C date", () => {
-			expect(processDate(new Date(2020, 6, 1, 0, 0, 0, 0))).toBe(
-				"2020-07-01T00:00:00Z",
+			expect(processDate(new Date("2020-07-01T02:00+02:00"))).toBe(
```

Comment:
I'm not really sure about these changes tests: wouldn't it be better to also have a test that uses the `Date` constructor with digits other than timestamps? What do you think? I mean, as a demonstration.

The same comment is valid for other tests you changed.

Reply:
The problem with using `new Date(2020, 6, 1, 0, 0, 0, 0)` is, that it's interpreted in the local timezone.

> The parameter values are all evaluated against the local time zone, rather than UTC.

See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#individual_date_and_time_component_values).

So if I run `new Date(2020, 6, 1, 0, 0, 0, 0).toISOString()` on my computer, I get `2020-06-30T22:00:00.000Z`.

If someone in Sydney runs this, the result will be `2020-06-30T14:00:00.000Z`.

A possible solution would be to use [`Date.UTC()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC).

So the tests could look like this:
```ts
expect(processDate(new Date(Date.UTC(2020, 6, 1)))).toBe(
	"2020-07-01T00:00:00.000Z",
);
```

Reply:
Okay cool. No need for further tests!